src/mlnet/mlnet.csproj

```diff
@@ -3,11 +3,8 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.1</TargetFramework>
-    <PackAsTool>true</PackAsTool>
     <RootNamespace>Microsoft.ML.CLI</RootNamespace>
-    <ToolCommandName>mlnet</ToolCommandName>
-    <PackageId>mlnet</PackageId>
-    <Authors>mlnet</Authors>
+    <IncludeInPackage>mlnet</IncludeInPackage>
```

Comment:
How does this work? Since <PackAsTool>true</PackAsTool> and <ToolCommandName>mlnet</ToolCommandName> have been removed from here. Does <IncludeInPackage>mlnet</IncludeInPackage> get mlnet.nupkgproj?

Reply:
You don't want to put `<IncludeInPackage>mlnet</IncludeInPackage>` here.  This tells the packaging system to place the output of this project in the `lib` folder of the NuGet package. But we don't want that because this is a CLI tool. We want `mlnet.dll` to go into the `tools` folder.

Here is where `IncludeInPackage` gets respected:

https://github.com/dotnet/machinelearning/blob/c587f82b33e31e02c2ff817082716428c1d48c2e/src/Directory.Build.targets#L8-L15

You can see it only supports the `lib` folder.